src/modules/sensors/sensors.cpp

```diff
@@ -2070,6 +2070,11 @@ Sensors::task_main()
 	/* start individual sensors */
 	int ret = 0;
 
+/*
```

Comment:
This change would preclude configs that did use DriverFramework drivers being used for a qurt build (there is more than one qurt build).